cli/ops.rs

```diff
@@ -498,10 +499,30 @@ fn op_fetch_module_meta_data(
   let use_cache = !state.flags.reload;
   let no_fetch = state.flags.no_fetch;
 
+  // TODO(bartlomieju): I feel this is wrong - specifier is only resolved if there's an
```

Comment:
What does the spec say about modules that don't appear in the import map?
Is there some default behavior to fall back to?

Reply:
Yes, it's fall-through - in our case `ImportMap::resolve` returns None and specifier is resolved as if there was no import map

Reply:
But my comment concerns whole `op_fetch_module_meta_data` - I believe that there should be resolving of specifier even if there is no import map - to see that, please tak a look at `tests/error_004_missing_module.ts`. It not returning proper error. More for reference in this [comment](https://github.com/denoland/deno/pull/2360#issuecomment-499165271)